src/reconcile/rec_write.c

```diff
@@ -1394,8 +1395,8 @@ __rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins,
 		r->max_txn = max_txn;
 
 	/* Update the maximum timestamp. */
-	if (first_ts_upd != NULL && r->max_timestamp < first_ts_upd->start_ts)
-		r->max_timestamp = first_ts_upd->start_ts;
+	if (first_ts_upd != NULL && r->max_timestamp < first_ts_upd->durable_ts)
+		r->max_timestamp = first_ts_upd->durable_ts;
```

Comment:
Why did you choose to switch to the durable timestamp here?

Reply:
As discussed offline, it is OK to switch to durable timestamp here. Taken a look at all the places where `max_timestmap` can have an effect. This is a behavior change and can have some knock-on effect in eviction